go.mod

```diff
@@ -85,6 +85,12 @@ replace (
 	// yet provides the same fix.
 	github.com/docker/docker => github.com/docker/docker v1.4.2-0.20191121165722-d1d5f6476656
 
+  // Temporary prebase rc.1 pins
+  github.com/openshift/api => github.com/openshift/api prebase-1.19-rc.1
+  github.com/openshift/client-go => github.com/openshift/client-go prebase-1.19-rc.1
+  github.com/openshift/library-go => github.com/openshift/library-go prebase-1.19-rc.1
+  github.com/openshift/apiserver-library-go => github.com/damemi/apiserver-library-go prebase-1.19-rc.1
```

Comment:
I couldn't find a merged `rc.1` branch (only one for `rc.2`: https://github.com/openshift/apiserver-library-go/pull/31), so note that I forked it and re-did the rc2 bump with an rc1 bump. There was only 1 other commit so for a prebase I think this is ok